swift/ios-application/build.gradle

```diff
@@ -0,0 +1,26 @@
+buildscript {
+    dependencies {
+        classpath "org.gradle.samples:ios-plugin:latest.integration"
+    }
+}
+
+plugins {
+    id 'swift-application'
+    id 'xcode'
+}
+
+apply plugin: 'org.gradle.samples.ios-application'
+
+group = "org.gradle.samples"
+
+// Add iOS specific resource to Xcode IDE project
+file('src/main/resources/Base.lproj').listFiles().each {
+    xcode.project.groups.sources.from it
+}
+xcode.project.groups.sources.from 'src/main/resources/Info.plist'
+xcode.project.groups.sources.from 'src/main/resources/Assets.xcassets'
+
+// Configure iOS specific task source location
+compileStoryboard.sources.from file('src/main/resources/Base.lproj').listFiles()
```

Comment:
This should also be using a file collection, same as above (the same file collection, given that they're the same files).